caikit/runtime/servicers/global_train_servicer.py

```diff
@@ -59,20 +55,8 @@ class GlobalTrainServicer:
     def __init__(self, training_service: ServicePackage):
         self._training_service = training_service
         self._model_manager = ModelManager.get_instance()
```

Comment:
Not related to this PR but this class really needs to get renamed

Reply:
oof, actually this does pull in the `MODEL_MANAGER` now too so it is a bit confusing to see `ModelManager` and `MODEL_MANAGER` and have them be completely different things.

`ModelManager` -> `RuntimeModelCache`?

Reply:
Ug, yeah, I really don't like this and it would be good go fix. Then again, I'm not sure I want to rock the boat that much (and I _just_ started another branch that touches `runtime/model_management/model_manager.py` 😢)

Reply:
I'm going to hold on this since I think renaming this would be an additional big change and it _might_ intersect with some of our composition/orchestration discussions.